x-pack/plugins/ml/public/application/components/anomaly_detection_empty_state/anomaly_detection_empty_state.tsx

```diff
@@ -0,0 +1,73 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React, { FC } from 'react';
+import { FormattedMessage } from '@kbn/i18n-react';
+import { EuiButton, EuiEmptyPrompt, EuiImage } from '@elastic/eui';
+import adImage from './ml_anomaly_detection.png';
+import { ML_PAGES } from '../../../../common/constants/locator';
+import { useMlLocator, useNavigateToPath } from '../../contexts/kibana';
+import { checkPermission } from '../../capabilities/check_capabilities';
+import { mlNodesAvailable } from '../../ml_nodes_check';
+
+export const AnomalyDetectionEmptyState: FC = () => {
+  const disableCreateAnomalyDetectionJob = !checkPermission('canCreateJob') || !mlNodesAvailable();
+
+  const mlLocator = useMlLocator();
+  const navigateToPath = useNavigateToPath();
+
+  const redirectToCreateJobSelectIndexPage = async () => {
+    if (!mlLocator) return;
+    const path = await mlLocator.getUrl({
+      page: ML_PAGES.ANOMALY_DETECTION_CREATE_JOB_SELECT_INDEX,
+    });
+    await navigateToPath(path, true);
+  };
+
+  return (
+    <EuiEmptyPrompt
```

Comment:
I think it would be nice to add a link to the Anomaly detection docs at the bottom of this prompt, similar to what's done in Data view management. It could link to the [overview docs page](https://www.elastic.co/guide/en/machine-learning/current/ml-overview.html) perhaps?

![image](https://user-images.githubusercontent.com/7405507/150776847-7540fe0f-4f89-4098-a448-a5f64cb39814.png)

Also what do you think of the styling used in the Data views prompt - with the filled background and no border?

Reply:
Updates styles in https://github.com/elastic/kibana/pull/123462/commits/c378bdf73f241cce60cbdea44ce4bd0252ebc0a6 and added a footer with docs link (target is https://www.elastic.co/guide/en/machine-learning/master/ml-ad-overview.html) in https://github.com/elastic/kibana/pull/123462/commits/fc1712071818082de2fa295b33e0fd342e70a732